modules/mono/SdkPackageVersions.props

```diff
@@ -0,0 +1,6 @@
+<Project>
+  <PropertyGroup>
+    <PackageVersion_Godot_NET_Sdk>4.0.0-dev4</PackageVersion_Godot_NET_Sdk>
+    <PackageVersion_Godot_SourceGenerators>4.0.0-dev1</PackageVersion_Godot_SourceGenerators>
```

Comment:
Is it intended to have version `4.0.0-dev4` instead of `4.0.0-dev1`?

Reply:
Yes, since these are the versions of each package, dev1 to dev3 have already been used. It does seem a bit strange that they're out of sync, so maybe the source generators package should just start at `4.0.0-dev4`.

Reply:
For stable releases they should be the same version, but during development there will be new additions to the source generator while the Sdk may remain the same. Technically we can release an unchanged new version of the Sdk to keep the version in sync with the other package, but that doesn't feel right. The only case where that would be appropriate is when going from rc to stable without changes in the packages.